items/item.go

```diff
@@ -4,27 +4,30 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"log"
 	"strconv"
 	"strings"
+	"sync"
 
 	"github.com/ndlib/bendo/store"
 )
 
 // A Store holds a collection of items
 type Store struct {
-	cache ItemCache
-	S     store.Store // the underlying bundle store
+	cache    ItemCache
+	S        store.Store // the underlying bundle store
+	useStore bool        // true - use bundlestore: false - use only itemCache
```

Comment:
I thought of that. Noted.

On Thu, Aug 25, 2016 at 12:22 PM, Don Brower <notifications@github.com>
wrote:

> In items/item.go
> <https://github.com/ndlib/bendo/pull/136#discussion_r76276515>:
>
> >
> >  	"github.com/ndlib/bendo/store"
> >  )
> >
> >  // A Store holds a collection of items
> >  type Store struct {
> > -	cache ItemCache
> > -	S     store.Store // the underlying bundle store
> > +	cache    ItemCache
> > +	S        store.Store // the underlying bundle store
> > +	useStore bool        // true - use bundlestore: false - use only itemCache
>
> FYI, useStore needs a mutex to serialize access. But since it will mostly
> be read, I don't think we need to add it right now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ndlib/bendo/pull/136/files/116b45889c785d5598950eba80d6730d33f98945#r76276515>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADNGTJYeAeA4bbhoSZk3FGG6_C-jNLHcks5qjcFTgaJpZM4JsckT>
> .
>